src/llmq/context.h

```diff
@@ -27,25 +27,26 @@ class CInstantSendManager;
 }
 
 struct LLMQContext {
+    LLMQContext() = delete;
+    LLMQContext(const LLMQContext&) = delete;
     LLMQContext(CEvoDB& evo_db, CTxMemPool& mempool, CConnman& connman, CSporkManager& sporkman,
                 const std::unique_ptr<PeerManager>& peerman, bool unit_tests, bool wipe);
     ~LLMQContext();
 
-    void Create(CEvoDB& evo_db, CTxMemPool& mempool, CConnman& connman, CSporkManager& sporkman,
-                const std::unique_ptr<PeerManager>& peerman, bool unit_tests, bool wipe);
-    void Destroy();
     void Interrupt();
     void Start();
     void Stop();
 
+    // Guaranteed if LLMQContext is initialized - all members are valid too
     std::shared_ptr<CBLSWorker> bls_worker;
     std::unique_ptr<llmq::CDKGDebugManager> dkg_debugman;
+    llmq::CQuorumBlockProcessor* quorum_block_processor;
```

Comment:
Minor nit, previous order of segregating containerized pointers from raw pointers was preferable imo

Reply:
raw quorum_block_processor should be initialilzed before other members, order of initialization of members is theirs order.

Reply:
While constructors are the last in the order of initialization, the member's true values aren't initialized inline but within the constructor itself, the few inline values we set are `nullptr`s anyway. 

If it was something along the lines of 
```
std::unique_ptr<CExampleClass> uptr_xmpl{std::make_unique<CExampleClass>()};
``` 
as the definition and the constructor dereferences `uptr_xmpl` then this approach would be the best, but as this isn't the case and mirroring the order of initialization in the constructor with the definition isn't a technical requirement, having it as more readable is much preferred.

But again, this is my preference talking and is not a dealbreaker.

Reply:
> While constructors are the last in the order of initialization

Not sure if I got correctly this question.  Constructors of members are called in the order of declaration, actually?

 - https://stackoverflow.com/questions/10404889/gcc-lack-of-warning-about-order-of-initialization-in-constructors
 - C++ Standard 12.6.2 - quote is here: https://stackoverflow.com/questions/2669888/initialization-order-of-class-data-members
> — Then, nonstatic data members shall be initialized in the **order they were declared in the class definition** (again regardless of the order of the mem-initializers).


Btw, I refactored this part, please, check.